app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,20 @@
+class SessionsController < ApplicationController
+  def new
+  end
+
+  def create
+    user = User.find_by email: params[:session][:email].downcase
+    if user && user.authenticate(params[:session][:password])
```

Comment:
Have you tried removing optional bracket () and see the result ?

Reply:
@mdmahmudurrahman I tried, it was error.

Reply:
You can try with
```
user&.authenticate params[:session][:password]
```